sdk/EAVFW.Extensions.Manifest.SDK.csproj

```diff
@@ -20,7 +20,7 @@
 
 
 	<ItemGroup>
-
+		<PackageReference Include="System.Text.Json" Version="8.0.0" />
```

Comment:
This will force all users of eavfw to depend of system.text.json 8.0.0. Dont know if thats an issue. We can always go with it and go back and lower the version if its an issue.

Reply:
It was required when I tried the polymorphic deserialisation, but I don't do it anyways so I'll remove it :)